runtime/test-runtime/src/lib.rs

```diff
@@ -311,7 +311,7 @@ impl staking::Trait for Runtime {
 	type ElectionLookahead = ElectionLookahead;
 	type Call = Call;
 	type UnsignedPriority = StakingUnsignedPriority;
-	type MaxIterations = MaxIterations;
+	// type MaxIterations = MaxIterations;
```

Comment:
There's commented out at multiple places

Reply:
I think you need to merge polkadot/master